js/components/Card.js

```diff
@@ -48,8 +48,9 @@ class Card extends React.Component {
          let toValue = 0;         
          let distanceToBoard = styles.cardHeight + styles.cardHeight;
          if(Math.abs(this.state.pan.y._value) >= distanceToBoard) {
-           toValue = -distanceToBoard;
-           if (!this.props.inPlay) {
+           if (!this.props.inPlay 
+               && this.props.playerMana >= this.props.cost) {
```

Comment:
i think for multi-line booleans it is nicer to put the && on the end of a line, because that way the text (like this.props) for similar things will line up. super small detail tho